ouroboros-network-framework/src/Simulation/Network/Snocket.hs

```diff
@@ -85,13 +88,50 @@ stepScript :: MonadSTM m => LazySTM.TVar m (Script a) -> m a
 stepScript = atomically . stepScriptSTM
 
 
+data Connection m = Connection
+    { -- | Attenuated channels of a connection.
+      --
+      connChannelLocal  :: !(AttenuatedChannel m)
+    , connChannelRemote :: !(AttenuatedChannel m)
+
+      -- | SDU size of a connection.
+      --
+    , connSDUSize       :: !SDUSize
+    }
+
+
+-- | Connection id independend of who provisined the connection. 'NormalisedId'
+-- satisfies the invariant that for @NormlisedId a b@ we have @a <= b@.
```

Comment:
@NormlisedId a b@ -> @NormalisedId a b@

Reply:
here 'a' and 'b' refer to `nidLow` and `nidHigh`, because `NormalisedId` only has 1 type parameter?

Reply:
Yes. It's actually better to use `nidLow` and `nigHigh` in the haddocks.